index.js

```diff
@@ -0,0 +1,2 @@
+const NombreUsuarior=prompt('Ingrese su Nombre');
+alert(`hello ${NombreUsuarior}!`);
```

Comment:
En este caso me parece mejor que no uses interpolacion de strings
```suggestion
alert(`hello` + NombreUsuarior);
```